storage/innobase/btr/btr0btr.cc

```diff
@@ -191,7 +191,7 @@ bool btr_root_fseg_validate(ulint offset, const buf_block_t &block,
   sql_print_error("InnoDB: Index root page " UINT32PF " in %s is corrupted "
                   "at " ULINTPF,
                   block.page.id().page_no(),
-                  UT_LIST_GET_FIRST(space.chain)->name);
+                  UT_LIST_GET_FIRST(space.chain)->name, offset);
```

Comment:
I guessed this one as well and could use a contact.

Reply:
I had it extracted as a part of #3493 but did not file a separate ticket.

Reply:
This seems to be a follow-up fix of 0b47c126e31cddda1e94588799599e138400bcf8. It obviously needs to be filed as such, targeting the 10.6 branch. I’d also simplify the format string:
```c++
  const fil_node_t *const node= UT_LIST_GET_FIRST(space.chain);
  sql_print_error("InnoDB: Index root " UINT32PF " in %s is corrupted at %zu",
                  block.page.id().page_no(), node->name, offset);
```
By removing the word "page" from this hopefully never seen message, we can fit it in a single line.

Reply:
> By removing the word "page" from this hopefully never seen message, we can fit it in a single line.

I managed to just fit the line without removing the word by replacing `UINT32PF` with its C/C++-standard one.

```cc
  sql_print_error("InnoDB: Index root page %"PRIu32" in %s is corrupted at %zu",
```

I’ll amend this to #3493 in a movement along with a rebase.

(For passersby: See [this reply thread](#discussion_r1703530824) re. `%zu`)

Reply:
Nevermind.

> C++11 requires a space between literal and string macro

With this being the case, I’m not interested in squeezing the format string.
Removing a word would be inconsitent with prior versions.
Reformatting the lines creates unnecessary diffs and can instead go with [the replacement of `UINT32PF` and `ULINTPF`](#discussion_r1833782627).

I’m undoing.

Reply:
> and I also think this should go into 10.6 with its own MDEV.

I have included this in [MDEV-35430](https://jira.mariadb.org/browse/MDEV-35430).